app/src/main/java/org/apache/fineract/data/remote/BaseUrl.java

```diff
@@ -10,8 +10,8 @@
  */
 public class BaseUrl {
 
-    public static final String PROTOCOL_HTTPS = "https://";
-    public static final String API_ENDPOINT = "pilot.kuelap.io";
+    public static final String PROTOCOL_HTTPS = "http://";
+    public static final String API_ENDPOINT = "buffalo.mifos.io:4200";
```

Comment:
@jawidMuhammadi  Which baseUrl should I use? Both the older and updated URLs are not working.